src/jetstream/mod.rs

```diff
@@ -371,9 +374,13 @@ pub enum ErrorCode {
     ConsumerDeliverCycle = 10081,
     /// Consumer requires ack policy for max ack pending
     ConsumerMaxPendingAckPolicyRequired = 10082,
+    /// JSConsumerMaxRequestBatchNegativeErr consumer max request batch needs to be > 0
+    JSConsumerMaxRequestBatchNegativeErr = 10114,
```

Comment:
Nit, for consistency

```suggestion
    JSConsumerMaxRequestBatchNegative = 10114,
```